sql/rpl_parallel.cc

```diff
@@ -50,6 +61,9 @@ rpt_handle_event(rpl_parallel_thread::queued_event *qev,
   err= apply_event_and_update_pos_for_parallel(ev, thd, rgi);
 
   thread_safe_increment64(&rli->executed_entries);
+#ifdef WITH_WSREP
+      wsrep_after_statement(thd);
```

Comment:
Fix indentation.

How about error handling? At least warning should be logged if the `wsrep_after_statement()` fails here.